diff --git a/node_modules/react-native-get-sms-android/android/src/main/java/com/react/SmsModule.java b/node_modules/react-native-get-sms-android/android/src/main/java/com/react/SmsModule.java
index 7a27616..c7486e4 100644
--- a/node_modules/react-native-get-sms-android/android/src/main/java/com/react/SmsModule.java
+++ b/node_modules/react-native-get-sms-android/android/src/main/java/com/react/SmsModule.java
@@ -241,8 +241,8 @@ public class SmsModule extends ReactContextBaseJavaModule /*implements LoaderMan
             ArrayList<PendingIntent> sentPendingIntents = new ArrayList<PendingIntent>();
             ArrayList<PendingIntent> deliveredPendingIntents = new ArrayList<PendingIntent>();
 
-            PendingIntent sentPI = PendingIntent.getBroadcast(context, 0, new Intent(SENT), 0);
-            PendingIntent deliveredPI = PendingIntent.getBroadcast(context, 0, new Intent(DELIVERED), 0);
+            PendingIntent sentPI = PendingIntent.getBroadcast(context, 0, new Intent(SENT), PendingIntent.FLAG_IMMUTABLE);
+            PendingIntent deliveredPI = PendingIntent.getBroadcast(context, 0, new Intent(DELIVERED), PendingIntent.FLAG_IMMUTABLE);
 
             //---when the SMS has been sent---
             context.registerReceiver(new BroadcastReceiver() {
